ui/jstests/test-static-assets.jsx

```diff
@@ -0,0 +1,159 @@
+define(['QUnit', 'jquery', 'static_assets', 'reactaddons',
+  'test_utils', 'jquery_mockjax'], function(
```

Comment:
I don't think you need `jquery_mockjax` here, we do all that through `TestUtils` now. I should remove it from `test-learning-resource.jsx` too